src/app/client/types.ts

```diff
@@ -39,3 +39,23 @@ export interface IKubeStatus extends IMetaTypeMeta {
   };
   metadata: Partial<IMetaObjectMeta>;
 }
+
+export type mustGatherStatus = 'error' | 'inprogress' | 'completed' | 'new';
+
+export interface IMustGatherResponse {
+  ['archive-name']: string;
```

Comment:
Well actually in that case I found out that not using the brackets is valid too. Up to you if you'd like.

Reply:
I can drop the brackets for the definitions, mainly just the property access that requires it.